.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: PyPi release
+run-name: CI pipeline
+on:
```

Comment:
Can have multiple options here, each with a different order?
Option 1: when a pull request is done for feature branch -->> master. In this scenario keep the order as is (style, build, test, documentation).  
Option 2: local machine commits to feature branch (local -->> feature branch on github). In this scenario it would be ideal if the order can be different (test, build, style, documentation)

Reasoning for this:
If we work on a branch like KPI_prototyping and I make commits to the branch during dev, we would like to focus on the code functionality first, then finer detailed formatting issues will be resolved once the code functions the way it should. In such a case we would like the pipeline to 1st do the test, then build, then style and then documentation

Reply:
On Github these jobs are executed in parallel. In other words, each job is started at the same time. Does the currently implementation already solve this for you?

Reply:
Oh nice, and yes! Yes. Because in gitlab the entire pipeline will stop if the 1st step fails. Then the test part is never started.